test/tests/functional/cgroups_tests.py

```diff
@@ -52,7 +52,7 @@
                     "eatmem.c"]
 
 
-class CgroupsTests(PBSTestSuite):
+class TestCgroups(TestFunctional):

```

Comment:
@hirenvadalia There is an already existing ticket for the same - PP-414:
PP-414: cgroup test file does not follow PTL naming conventions